### DISCUSSION TITLE
Awesome library, but does not have datetime picker?

### DISCUSSION THREAD

**Discussion by 5startsolutions** [chakra-ui/chakra-ui#4932] [Ideas] [answered]
I really enjoy this library the only thing is that it doesn't have a datetime picker, hopefully u incorpore it soon :)

**Comment by daniel-schulz-codes** [comment, marked as answer]
Hej @5startsolutions thanks for reaching out -  we currently don't provide a datetime-picker, but there are workarounds available.

This can be a starting point for further research (https://github.com/chakra-ui/chakra-ui/issues/580) 

Currently, there are two approaches documented:

1. styling `react-datepicker` (https://github.com/Hacker0x01/react-datepicker) with a chakra style
2. use `dayzed`https://github.com/deseretdigital/dayzed a headless datepicker and implement the UI complete with chakra

**Comment by 5startsolutions** [comment]
Wow, thank u very much
